packages/zend-form/library/Zend/Form/Element/File.php

```diff
@@ -122,7 +122,7 @@ public function setPluginLoader(Zend_Loader_PluginLoader_Interface $loader, $typ
      */
     public function getPluginLoader($type)
     {
-        $type = strtoupper($type);
+        $type = is_string($type) ? strtoupper($type) : $type;
```

Comment:
idem

Reply:
I will leave this one as it calls `parent::getPluginLoader($type)` and potentially that could be called with `null` and handled there.

Reply:
ok, thank you for taking care about BC 👍